draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -346,13 +457,14 @@ The RSASSA-PSS parameters, defined as in {{!RFC8017, Section 9.1.1}}, are as fol
 - MGF: mask generation function
 - sLen: intended length in octets of the salt
 
-Implementations MUST support PS384-encoding, using SHA-384 as Hash and MGF functions
-and sLen = 48, as described in {{!RFC8230, Section 2}}. It is RECOMMENDED that
-implementations also support encoding using SHA-384 as Hash and MGF functions and
-sLen = 0. Note that setting sLen = 0 has the result of making the signature
-deterministic.
+It is RECOMMENDED that implementations support the following encoding options:
+
+- SHA-384 as Hash and MGF functions and sLen = 48, as described in {{!RFC8230, Section 2}}; and
+- SHA-384 as Hash and MGF functions and sLen = 0.
+
+Note that setting sLen = 0 has the result of making the signature deterministic.
```

Comment:
Shall we keep supporting deterministic signatures? In that case, the input must have high entropy and the message salt must be empty in addition to having `sLen=0`.

Reply:
Indeed, this is the question I asked in the PR description. That seems to be the key question we need to answer. The outcome will become clear based on that. 

To be clear, I don't have a use case for deterministic signatures right now, so I would be OK removing support. But I know it was registered as an important use case to some people during the adoption process of this draft.